Octokit.Tests/Clients/StarredClientTests.cs

```diff
@@ -346,19 +345,31 @@ public class TheGetAllForUserMethod
         public class TheGetAllStargazersForRepoMethod
         {
             [Fact]
-            public void RequestsCorrectUrl()
+            public async Task RequestsCorrectUrl()
```

Comment:
Is this required?  I had a look at most `RequestsCorrectUrl()` tests and they aren't `async Task<>` they are just `void`